deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py

```diff
@@ -295,3 +295,12 @@ def check_refresh_feed_log(provider_name):
     check_vuln_detector_event(timeout=vd.VULN_DETECTOR_GLOBAL_TIMEOUT,
                               callback=f"Refresh of '{provider_name}' database finished",
                               error_message=f"Could not find {provider_name} feed refresh log")
+
+
+def check_configuration_error():
+    """Check the configuration error event in ossec.log"""
+    check_vuln_detector_event(timeout=vd.DOWNLOAD_TIMEOUT,
+                              callback=r".* \(\d+\): Configuration error at.*",
+                              error_message=f"Could not find the event 'Configuration error at "
+                                            f"'etc/ossec.conf' in ossec.log",
+                              prefix='.*wazuh-modulesd.*')
```

Comment:
It is necessary to set the prefix?

Reply:
If I do not set the prefix, the event log is not catched